docs/frameworks/nitrojs.md

```diff
@@ -0,0 +1,246 @@
+---
+description: Learn how to deploy a Nitro.js application with Genezio.
+---
+
+import Tabs from '@theme/Tabs';
+
+import TabItem from '@theme/TabItem';
+
+# Nitro.js
+
+<head>
+    <title>Nitro.js | Genezio Documentation</title>
+</head>
+
+Nitro.js is a modern framework for building server-side applications, designed to optimize performance and streamline development.
+It offers a comprehensive set of features for creating web servers and APIs, supporting multiple deployment targets. 
+
+# Deployment
+
+In this tutorial, you will learn how to deploy an existing Nitro.js app using Genezio, a serverless deployment platform that simplifies app management and reduces costs. We'll cover the benefits, answer common questions, and provide detailed deployment steps.
+
+## Why Use Genezio?
+
+### Benefits of Using Genezio
+
+1. **Faster Deployment**: Genezio optimizes the deployment process, significantly reducing the time it takes to get your app live.
+2. **Lower Costs**: Pay only for the resources you use, thanks to Genezio’s efficient infrastructure and scalable solutions.
+3. **Reliable and Scalable**: No longer worry about PM2 and Docker. You write the code, and Genezio seamlessly scales it across multiple cores.
+4. **Enhanced Security**: Built-in security features and automatic updates protect your app against the latest threats without additional effort.
+5. **Reduced Management Overhead**: Focus on developing your app instead of managing servers and routine maintenance tasks.
+6. **Easy Migration**: Obtain all these advantages with minimal code changes.
+
+### How Does Genezio Handle High Traffic?
+
+Genezio, being a Function as a Service (FaaS) platform, automatically scales your application based on traffic demands. It works by executing functions in response to events and can handle a large number of requests concurrently. This ensures your app remains responsive and performs well during traffic spikes.
+
+<!--
+### What Kind of Support Does Genezio Offer?
+
+Genezio provides 24/7 support for paid plans to assist with any questions or issues. Our dedicated support team ensures your app runs smoothly on our platform. -->
+
+### Do I Need to Modify My Nitro App Code?
+
+Migrating your Nitro app to Genezio typically requires minimal changes. Our migration guide provides detailed instructions for necessary compatibility adjustments.
+
+## Prerequisites
+
+### 1. Install genezio
+
+Use your preferred package manager to install genezio:
+
+<Tabs>
```

Comment:
Add the same `groupId` to all the Tabs that contain `npm`, `pnpm` and `yarn`. This way, after the user changes the tab to `yarn` once, the other tab groups will be changed to `yarn` as well.

Take a look here: https://github.com/Genez-io/genezio-documentation/blob/main/docs/features/backend-deployment.md?plain=1#L32